ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryRoutingFunctionalTest.java

```diff
@@ -336,8 +336,6 @@ public void shouldFilterLaggyServers() {
     KsqlErrorMessage errorMessage = makePullQueryRequestWithError(clusterFormation.router.right,
         sql, LAG_FILTER_25);
     Assert.assertEquals(40001, errorMessage.getErrorCode());
-    Assert.assertTrue(errorMessage.getMessage()
```

Comment:
whats the new error? could we assert on that still

Reply:
I added this back.  I was originally trying to mitigate some periodic flakiness, but I fixed that elsewhere.